packages/@glimmer/syntax/lib/generation/printer.ts

```diff
@@ -245,7 +245,7 @@ export default class Printer {
   }
 
   CloseElementNode(el: ASTv1.ElementNode): void {
-    if (el.selfClosing || voidMap[el.tag.toLowerCase()]) {
+    if (el.selfClosing || voidMap[el.tag]) {
```

Comment:
I don't think this really matches Glimmer's rule about component naming, which are based solely on whether the first letter is capital. `<lINK>` is still a void tag.

Something like

```js
function isVoidTag(tag: string): boolean {
  return voidMap[tag.toLowerCase()] && tag[0].toLowerCase() === tag[0];
}

CloseElementNode(el: ASTv1.ElementNode): void {
     if (el.selfClosing || isVoidTag(el.tag)) {
```

Reply:
it doesn't look like `<lINK>` is supported even before these changes -- errors occur elsewhere.
I do like this function tho, so I'll add this.

Reply:
pushed